### ISSUE TITLE
Happy Birthday!

### ISSUE THREAD

**Issue by EliCDavis** [EliCDavis/resistance-telegram-bot#19]
Create a file named the id of the chat that the bot is apart of. 
File contents would contain birthdays of users in chat in the form of username and birthday

Example:
@Bo2s 11/28/1995
@blondeinblue 2/2/1995

On startup on the bot load in that file and create a sorted array in the form of the user who's birthday is next get's the first entry of the array. Each tick it checks if the first entry in the array's birthday is today, and on success, send a happy birthday message and shift their entry to the back of the array.

**Comment by JacobAMason** [comment]
Don't use years or make them optional in the file. If included, the bot can say how old they are, otherwise, it will just wish them a happy birthday.

**Comment by EliCDavis** [comment]
Very good point. Everyone 21+ at this point anyway year isn't significant.

**Comment by JacobAMason** [comment]
Yeah. It's just that I can see the file getting checked in on accident and it wouldn't be the best piece of information to leave public.
